Interrogator.xUnit/NotConcurrentAttribute.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using Interrogator.xUnit.Common;
+using Interrogator.xUnit.Utilities;
+
+namespace Interrogator.xUnit
+{
+	[AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
+	public class NotConcurrentAttribute : DependsOnAttribute
+	{
+		public enum ConcurrencyScope
+		{
+			Class,
+			Namespace,
```

Comment:
Do you think it's worth have a top level class option of some sort, so if you are nesting classes you can say this whole class tree is the scope. I'm not sure what to call it (ParentClass? RootClass?), but I think this would be good to have.

Reply:
I went with Class Hierarchy.